tables/a.sh

```diff
@@ -0,0 +1 @@
+sed -i -e "s/PASSWORD/$MYSQL_ROOT_PASSWORD/g" /docker-entrypoint-initdb.d/tab.sql
```

Comment:
Don't call something `a.sh` :smile:. Make the filename meaningful. Use a numeric prefix (`00-set-passwords.sh`) if you need to ensure ordering between the scripts.

We don't want to use the root password for an unprivileged account. One option would be to look for all environment variables named `*_PASSWORD` and replace matching names in `tab.sql`.  So in `tab.sql` use something like `FLOCX_MARKET_PASSWORD` and `FLOCX_PROVIDER_PASSWORD` instead of `PASSWORD`.